SETUP.md

```diff
@@ -1,9 +1,9 @@
-# Cloning manubot-rootstock to create a new manuscript
+# Cloning rootstock to create a new manuscript
```

Comment:
Do you want to still include `Manubot` here?  Maybe `Cloning Manubot Rootstock...`?